### PR TITLE
Add sweeps for remainder, fmod, minimum, maximum, logical_and eltwise ops, rename eltwise sweeps

### PR DESCRIPTION
**Pull request by amalbasaTT**
### Ticket
https://github.com/tenstorrent/tt-metal/issues/11512

### Problem description
Missing sweeps in new sweeps infrastructure for eltwise ops:
1. remainder
2. fmod
3. minimum
4. maximum
5. logical_and

Eltwise sweeps need renaming.

### What's changed
Added .py sweeps to new sweeps infrastructure for the above mentioned eltwise ops. 
Renamed all eltwise sweeps.
Modified .github/workflows/ttnn-run-sweeps.yaml file to include above mentioned eltwise ops.
Overloaded invalidate_vector function in sweeps which don't support Grayskull (or which only support Wormhole_B0).

### Checklist
- [x] ttnn sweeps CI passes (https://github.com/tenstorrent/tt-metal/actions/runs/11029909710)
- [x] Post commit CI passes
- [x] Blackhole Post commit (if applicable)
- [x] Model regression CI testing passes (if applicable)
- [x] Device performance regression CI testing passes (if applicable)
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

File: .github/workflows/ttnn-run-sweeps.yaml [hunk 1]
```diff
@@ -12,40 +12,45 @@ on:
...
-          - eltwise.binary.div.div_interleaved
-          - eltwise.binary.div_no_nan.div_no_nan_interleaved
-          - eltwise.binary.logical_or.logical_or_interleaved
-          - eltwise.composite.binary.addalpha.addalpha_interleaved
-          - eltwise.composite.binary.subalpha.subalpha_interleaved
-          - eltwise.ternary.addcmul.addcmul_interleaved
-          - eltwise.ternary.addcdiv.addcdiv_interleaved
-          - eltwise.ternary.mac.mac_interleaved
-          - eltwise.ternary.where.where_interleaved
+          - eltwise.unary.relu.relu
```
Comment (jdesousa-TT) [inline .github/workflows/ttnn-run-sweeps.yaml:15]: Is there a reason for all these eltwise files to be nested in folders? Are there plans to add more test modules in each one? If not can we make them top level?

File: tests/sweep_framework/sweeps/eltwise/binary/fmod/fmod.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        "input_a_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+        "input_b_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+        "output_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+    },
+}
+
+
+# TO-DO: Create an issue on this, since these constrictions are not mentioned in the documentation
+def invalidate_vector(test_vector) -> Tuple[bool, Optional[str]]:
+    if not is_wormhole_b0():
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/binary/fmod/fmod.py:45]: This is going to mean that all tests are invalid if you generate them on a system without a wormhole. This validation needs to be done in a device fixture on the runner side, not in the validation of the vectors.

File: tests/sweep_framework/sweeps/eltwise/binary/remainder/remainder.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        "input_a_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+        "input_b_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+        "output_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+    },
+}
+
+
+# TO-DO: Create an issue on this, since these constrictions are not mentioned in the documentation
+def invalidate_vector(test_vector) -> Tuple[bool, Optional[str]]:
+    if not is_wormhole_b0():
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/binary/remainder/remainder.py:57]: Same as above.

File: tests/sweep_framework/sweeps/eltwise/unary/ceil/ceil.py [hunk 4]
```diff
@@ -34,6 +34,12 @@
 }
 
 
+def invalidate_vector(test_vector) -> Tuple[bool, Optional[str]]:
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/unary/ceil/ceil.py:37]: Same as above.

File: tests/sweep_framework/sweeps/eltwise/unary/floor/floor.py [hunk 5]
```diff
@@ -35,6 +35,12 @@
 }
 
 
+def invalidate_vector(test_vector) -> Tuple[bool, Optional[str]]:
+    if not is_wormhole_b0():
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/unary/floor/floor.py:39]: Same as above

File: tests/sweep_framework/sweeps/eltwise/unary/frac/frac.py [hunk 6]
```diff
@@ -34,6 +34,13 @@
 }
 
 
+# TO-DO: Create an issue on this, since these constrictions are not mentioned in the documentation
+def invalidate_vector(test_vector) -> Tuple[bool, Optional[str]]:
+    if not is_wormhole_b0():
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/unary/frac/frac.py:39]: Same as above

File: tests/sweep_framework/sweeps/eltwise/unary/trunc/trunc.py [hunk 7]
```diff
@@ -34,6 +34,12 @@
 }
 
 
+def invalidate_vector(test_vector) -> Tuple[bool, Optional[str]]:
+    if not is_wormhole_b0():
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/unary/trunc/trunc.py:38]: Same as above

**Review by jdesousa-TT — CHANGES_REQUESTED** [review]

File: tests/sweep_framework/sweeps/eltwise/binary/fmod/fmod.py (hunk 2 above)
Comment (amalbasaTT) [reply tests/sweep_framework/sweeps/eltwise/binary/fmod/fmod.py:45]: Understood

File: .github/workflows/ttnn-run-sweeps.yaml (hunk 1 above)
Comment (amalbasaTT) [reply .github/workflows/ttnn-run-sweeps.yaml:15]: I understood that that is the plan. I will double check it.

Comment (amalbasaTT) [reply .github/workflows/ttnn-run-sweeps.yaml:15]: For some of these ops there is a plan to make another test modules, in next PR i'll make the test_modules which don't have more use cases to be top level.

Comment (jdesousa-TT) [reply .github/workflows/ttnn-run-sweeps.yaml:15]: Thanks

**Review by jdesousa-TT — APPROVED** [review]

**Review by jdesousa-TT — CHANGES_REQUESTED** [review]
floor.py still has invalidation

**Review by jdesousa-TT — APPROVED** [review]

**Pull request merged**